src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let i = 0;
+
+  return function(x) {
+    return function(y) {
+      i++;
+
+      if (i >= 4 && (i % 2 === 0)) {
```

Comment:
```suggestion
      if (i > 3 && i % 2 === 0) {
```
it looks more obvious, also, you don't need extra parentheses